riscv/insns/lpad.h

```diff
@@ -0,0 +1,9 @@
+if (p->extension_enabled(EXT_ZICFILP)) {
+  if (ZICFILP_READ_XLPE(STATE.v, STATE.prv) && STATE.elp == elp_t::LP_EXPECTED) {
```

Comment:
Is elp_t:: required?

Reply:
`elp_t` is an `enum struct`, so I need this `elp_t::` to access its enumerators.